hyperapp.d.ts

```diff
@@ -0,0 +1,149 @@
+declare module "hyperapp" {
+  export interface VNodeData {
+    [key: string]: any
```

Comment:
Size doesn't matter, but I dont want you to be in charge of a monster. The react types is too too.

Here, I think we can allow the use of any. I dont want to maintain a complete list of HTML attribute, and how about custom ones ? And how about new ones ?
An attribute is nothing more a key and a value, and a value can be everything so `any`.

@nippur72 Do you agree ?